Update TASK_REQUEST.md with resource library link (#72)
Comet-Robotics/chessBot: .github/ISSUE_TEMPLATE/TASK_REQUEST.md

#### Relevant Links:
<!--

> Link the relevant resources that would be needed from the code base, or internet. (You can use the resource library to help search for relevant links)
> Here is a link to the resource library: https://github.com/Comet-Robotics/chessBot/wiki/Resource-Library

Reviewer: ```suggestion
Link the relevant resources that would be needed from the code base or internet.
You can use the resource library to help find relevant links: https://github.com/Comet-Robotics/chessBot/wiki/Resource-Library
```
